wellcomecollection/wellcomecollection.org
# Remove config redirects

## What does this change?

For #11348 

Removes the redirects from the next app, now that these are handled by the edge lambdas

## How to test

Need to restart the app when checking out the branch in order for the config to be applied

The following redirects should no longer work:
visiting localhost:3000/articles/the-power-of-art should render the page at localhost:3000/stories/the-power-of-art
visiting /articles takes you to /search/stories
visiting /stories/comic takes you to /search/stories filtered by comic

## How can we measure success?
N/A

## Have we considered potential risks?

We have to deploy the edge lambdas first so that they do the redirects. If the edge lambda redirects don't work correctly, then requests to /articles/... would fail. All our site links point to /stories/... not /articles/..., so the impact would be limited
